app/components/post_attachment_opengraph/post_attachment_opengraph.js

```diff
@@ -160,7 +116,8 @@ export default class PostAttachmentOpenGraph extends PureComponent {
 
     setImageSize = (imageUrl, originalWidth, originalHeight) => {
         if (this.mounted) {
-            const dimensions = calculateDimensions(originalHeight, originalWidth, this.getViewPostWidth());
+            const dimensions = calculateDimensions(originalHeight, originalWidth,
+                getViewPostWidth(this.props));
```

Comment:
can we use `getViewPortWidth` from the utils/images file instead? https://github.com/mattermost/mattermost-mobile/blob/master/app/utils/images.js#L65